src/app/utils/js-utils.ts

```diff
@@ -96,3 +96,9 @@ export function isBoolString(s: string) {
 export function snakeToCamelCased(str) {
     return str.replace(/-([a-z])/g, (g) => g[1].toUpperCase());
 }
+
+export function isUrl(path: string, location?: string) {
+    const regexp = /(ftp|http|https):\/\/(\w+:{0,1}\w*@)?(\S+)(:[0-9]+)?(\/|\/([\w#!:.?+=&%@!\-\/]))?/;
+
+    return regexp.test(path) ? path : `${location ? location + '/' : '' }${path}`;
```

Comment:
I don't like it like this:
- Why if the regex test doesn't pass it is ok to pass location+path? Who tells me that the string thus formed is correct?
- since the function name is a question, I expect to have a boolean as returning value.
I would leave just one param (`path`) and return the result of the test with the regex.
```
export function isUrl(path: string, location?: string) {
   return /(ftp|http|https):\/\/(\w+:{0,1}\w*@)?(\S+)(:[0-9]+)?(\/|\/([\w#!:.?+=&%@!\-\/]))?/.test(path);
}
```

Of course, you will have to write more code where you use it
```
s[AppConfig.evtSettings.files.editionImagesSource[key].value]
    ? {
        url: isUrl(s.corresp) ? s.corresp : localImagesFolder + s.corresp,
    } : {
        width: s[key][0].width,
        height: s[key][0].height,
        url: isUrl(s[key][0].url) ? s[key][0].url : localImagesFolder + s[key][0].url,
    });
```
but I think it is better that utils functions are as more generic as possible.